src/app/app.component.ts

```diff
@@ -512,4 +513,106 @@ export class AppComponent {
       }]
     }]
   };
+
+
+  // ----------------------------------------------------------------------
+  // Demo #5
+
+  chartLazyLoading: Highcharts.Options = {
+    chart: {
+      type: 'candlestick',
+      zoomType: 'x',
+      events: {
+        load: function() {
+          var chart = this;
+
+          $.getJSON('https://www.highcharts.com/samples/data/from-sql.php?callback=?', function(data) {
```

Comment:
Loading jQuery just for fetching a JSON is not optimal. How about `Highcharts.getJSON` from the data module instead of jQuery?

Reply:
After some research it turned out that jQuery is much easier in use for this particular example - the demo will remain as it is.